### ISSUE TITLE
Transfer file

### ISSUE THREAD

**Issue by MarcAntoine-Arnaud** [russelltg/srt-rs#215]
Hi @russelltg,

I'm looking to use that library to handle file transfer over SRT.
I have seen some code cannot work (Congestion mode `file` is not supported).

Is it possible to help me to find the way to implement it ?

Best regards,
Marc-Antoine

**Comment by robertream** [comment]
Yes, I can help guide you through this, but I'll have to warn you, it's a non-trivial change.

**Comment by russelltg** [comment]
My personal opinion is that file transfer is not a terribly compelling usecase for file transfer--my intuition is something like QUIC is better suited. 

Maybe there's something SRT provides here that is good though.
